### DISCUSSION TITLE
StreetComplete for cyclists

### DISCUSSION THREAD

**Discussion by apophenic-treehugger** [streetcomplete/StreetComplete#3241] [Ideas]
I have been thinking whether it is easy and smart to implement a **Cyclist mode** and have been inspired to hereby suggest it after reading the fruitful discussion and outcome of #3003.

Upon first activation, the user would be asked:
_"Only activate this mode when you're travelling by bike. (obviously, please don't use your phone while riding) Some quests cannot only be answered by actually surveying with a bike. Also, it is necessary that you have experience how it feels to ride different surfaces on different types of bicycles (road bikes, mountain bikes, etc.)."_
In case of a positive answer, a bicycle quests could be added:

- Add `smoothness` (was `surface` in the original version of this post, by mistake)
- Add `mtb:scale` on appropriate ways
- Add `incline` (with clear how-to instructions)
- other ideas?
These are a very valuable information when planning bike trips.

I know of at least one user who's very interested in a such mode (me).

**Comment by smichel17** [comment]
Something like this has been discussed before. I don't remember exactly where, but it was related to quest presets. Those discussions are listed under https://github.com/streetcomplete/StreetComplete/discussions/2789#discussioncomment-646707

**Comment by matkoniecz** [comment]
See https://github.com/streetcomplete/StreetComplete/issues/1850 for MTB scale quest

**Comment by riQQ** [comment]
See also https://github.com/streetcomplete/StreetComplete/discussions/2457#discussioncomment-268311 for more info on the mapping-as-a-cyclist use case.

**Comment by mnalis** [comment]
As others have mentioned, quest presets (available in SC [v34.0](https://github.com/streetcomplete/StreetComplete/releases/tag/v34.0)) allow you to create a "cycling preset", which only has quests you want. So, one only needs to suggest missing quests (one per issue) - but please read carefully https://github.com/streetcomplete/StreetComplete/wiki/Adding-new-Quests-to-StreetComplete about what must be satisfied for a quest to be eligible to be included.

- `surface` quest is already implemented, so you only  to enable it in "cycling preset"
- @matkoniecz mentioned an issue with suggestion for `mtb:scale` quest
- You also mention `incline` quest. As mentioned in a wiki above, it is likely problematic especially with regard to `Effort vs impact`, `No unanswerable quests` and `No spam`. Also see https://github.com/streetcomplete/StreetComplete/issues/2222 for problems measuring it if it is not spelled explicitly by street sign.
- riQQ mentioned other cycle-related quest that I would keep in my "cycling preset", also see https://github.com/streetcomplete/StreetComplete/issues/2565#issuecomment-778542281

**Reply by apophenic-treehugger** [reply]
Sorry, I meant `smoothness` rather than `surface`. Corrected that just now.

**Reply by mnalis** [reply]
@apophenic-treehugger we have an issue for that too: https://github.com/streetcomplete/StreetComplete/issues/1630
